tests/cmsis_rtos_v1/src/timer.c

```diff
@@ -97,3 +103,28 @@ void test_timer(void)
 	status = osTimerDelete(id2);
 	zassert_true(status == osOK, "error deleting periodic timer");
 }
+
+void test_timer_get_status(void)
+{
+	osTimerId id;
+	osStatus status;
+	u32_t timerDelay;
+
+	id = osTimerCreate(osTimer(Timer3), osTimerOnce, NULL);
+	zassert_true(id != NULL, "error creating one-shot timer");
+
+	/*Stop the timer before start */
+	status = osTimerStop(id);
+	zassert_true(status == osErrorResource, "error while stoping non active timer");
+
+	timerDelay = ONESHOT_TIME;
+	status = osTimerStart(id, timerDelay);
+	zassert_true(status == osOK, "error starting one-shot timer");
+
+	osDelay(timerDelay + 100);
+
```

Comment:
One more negative test: Stop an inactive timer and assert for `osErrorResource`

Reply:
@nniranjhana In line 117-118, test is stopping an inactive timer, and checking the status. When we create a timer using osTimerCreate(), it sets the timer status to NOT ACTIVE state. Timer status will move to ACTIVE state when we start the timer using  osTimerStart().

Reply:
@ajaykish Yes, you are right, sorry I missed that!